README.md

```diff
@@ -2,11 +2,24 @@
 
 An argument parser that is truly zero-cost, similar to getopts.
 
+## About
+
+`getargs` is a low-level, efficient and versatile argument parser that
+works similarly to `getopts`. It works by producing a stream of options,
```

Comment:
Since `getopts` is an actual crate, we should really specify that we're talking about POSIX sh's `getopts` command (or we could eliminate that comparison entirely).